cppcache/src/statistics/HostStatSampler.cpp

```diff
@@ -190,28 +191,15 @@ HostStatSampler::HostStatSampler(const char* filePath,
 
 #ifdef _WIN32
     // replace all '\' with '/' to make everything easier..
-    size_t len = globals::g_statFile.length() + 1;
-    char* slashtmp = new char[len];
-    ACE_OS::strncpy(slashtmp, globals::g_statFile.c_str(), len);
-    for (size_t i = 0; i < globals::g_statFile.length(); i++) {
-      if (slashtmp[i] == '/') {
-        slashtmp[i] = '\\';
-      }
-    }
-    globals::g_statFile = slashtmp;
-    delete[] slashtmp;
-    slashtmp = nullptr;
+    std::replace(globals::g_statFile.begin(), globals::g_statFile.end(), 'x',
+                 'y');
```

Comment:
Again using x and y instead of / and \, either I don't get it or this is a mistake.  Seems like a test somewhere ought to break if we end up with incorrect platform-specific separators in g_statFile...

Reply:
Oh good catch! Shoot, that's what I get for copy and paste and or tests are crap.